test/evm/hardhat/chain-adapters/Arbitrum_Adapter.ts

```diff
@@ -225,4 +253,55 @@ describe("Arbitrum Chain Adapter", function () {
       usdc.address
     );
   });
+  it("Correctly calls the OFT bridge adapter when attempting to bridge USDT", async function () {
+    const internalChainId = arbitrumChainId;
+
+    const { leaves, tree, tokensSendToL2 } = await constructSingleChainTree(usdt.address, 1, internalChainId, 6);
+    await hubPool
+      .connect(dataWorker)
+      .proposeRootBundle([3117], 1, tree.getHexRoot(), consts.mockRelayerRefundRoot, consts.mockSlowRelayRoot);
+    await timer.setCurrentTime(Number(await timer.getCurrentTime()) + consts.refundProposalLiveness + 1);
+
+    // set up correct messenger to be returned on a proper `oftMessengers` call
+    oftAddressBook.oftMessengers.whenCalledWith(usdt.address).returns(oftMessenger.address);
+
+    // set up `quoteSend` return val
+    const msgFeeStruct: MessagingFeeStructOutput = [
+      BigNumber.from(1e9 * 200_000), // nativeFee: 1 GWEI gas price * 200,000 gas cost
```

Comment:
These are big numbers and you're multiplying them as numbers, to be safer do: `toWeiWithDecimals("1", 9).mul(200_000)`